cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -825,7 +825,7 @@ def test_can_get_usage_info_when_special_characters_are_used(self):
                 {'id': 2, 'name': 'Group C', 'version': 1},
             ],
             'usage': [{
-                'url': '/container/{}'.format(vertical.location),
+                'url': '/container/i4x://org.25/course_25/split_test/Test_Content_Experiment_0JOS%C3%89_ANDR%C3%89S',
```

Comment:
I don't think this is the best way to do this. We weren't running into this problem before because the split test location was not in the url (it was the location of the group within the split test).

The actual results are:
0JOS%C3%89_ANDR%C3%89S
which I think is the URL encoding.

While the location of the split test is:
0JOS\xc3\x89 ANDR\xc3\x89S